virttest/utils_libvirt/libvirt_disk.py

```
@@ -160,6 +160,16 @@ def create_disk(
         )
 
 
+def _all_disk_targets():
+    devs = []
+    vmx = vm_xml.VMXML.new_from_dumpxml(vm_name)
```

Ruff (0.12.2)

Undefined name `vm_name`

(F821)